### PR TITLE
Update fr auth.json with translations for general_errors.

### LINKED ISSUE #32: Bug: French is failing schema validation due to missing values
### Prerequisites

- [X] I have searched the repository’s issues and [Kinde community](https://thekindecommunity.slack.com/archives/C04K316BXEH) to ensure my issue isn’t a duplicate
- [X] I have checked the latest version of the library to replicate my issue
- [X] I have read the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md)
- [X] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md)

### Describe the issue

Several keys missing from French translation file

### Library URL

https://github.com/kinde-oss/kinde-translations

### Library version

1

### Operating system(s)

macOS

### Operating system version(s)

N/A

### Further environment details

_No response_

### Reproducible test case URL

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by funkycharlie**
# Explain your changes

Fixes #32

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

🛟 _If you need help, consider asking for advice over in the [Kinde community](https://thekindecommunity.slack.com)._

### CODE REVIEW

**Review by DaveOrDead — APPROVED** [review]

File: fr/auth.json [hunk 1]
```diff
@@ -246,5 +246,11 @@
       "email_greeting": "Salut,",
       "email_disclaimer": "Vous avez reçu cet email car vous avez demandé un code de confirmation de ${business_name}"
     }
```
Comment (DaveOrDead) [inline fr/auth.json:248]: @funkycharlie currently the JSON breaks the validator due to a missing comma here

**Review by DaveOrDead — CHANGES_REQUESTED** [review]
Hey @funkycharlie - massive thanks for the PR :) Just one tiny change and I think we're good to go

File: fr/auth.json (hunk 1 above)
Comment (funkycharlie) [reply fr/auth.json:248]: I think that was already there before, I haven't touched those parts. But yeah that's all good

**Comment by funkycharlie** [comment]
Hi @DaveOrDead, do I have to change the file myself or is it done, and i only had to press resolve? I'm pretty new to this

**Comment by DaveOrDead** [comment]
@funkycharlie apologies I missed this - it was a request for you to make the change :)

**Comment by DanielRivers** [comment]
@funkycharlie If you add that comma I will get this merged, you didn't enable maintainer contributions so I can't make the change.

**Comment by DaveOrDead** [comment]
Closing due to inactivity - thanks all :)

**Pull request closed without merging**